osu.Game/Database/StandardisedScoreMigrationTools.cs

```diff
@@ -367,8 +367,8 @@ public static long ConvertFromLegacyTotalScore(ScoreInfo score, LegacyBeatmapCon
 
                 case 3:
                     return (long)Math.Round((
-                        990000 * comboProportion
-                        + 10000 * Math.Pow(score.Accuracy, 2 + 2 * score.Accuracy)
+                        850000 * comboProportion
+                        + 150000 * Math.Pow(score.Accuracy, 2 + 2 * score.Accuracy)
```

Comment:
Given that this is in `ConvertFromLegacyTotalScore()`, should we be bumping score version again to upgrade previously-imported stable scores to the new proportions?

(Key point, I am not talking about *lazer* scores here; emphasis on legacy total score.)

Reply:
Might as well, yeah.